### ISSUE TITLE
[4.0] Postinstall messages: External links are not accessible

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#34669]
### Steps to reproduce the issue
Open Postinstall messages.

### Expected result
Extermal links have the External-link symbol. 

### Actual result
![grafik](https://user-images.githubusercontent.com/1035262/124011899-ccfd8300-d9e0-11eb-9491-66964c5a79c3.png)

Links must be underlined or must have the target="blank" attribute

**Comment by brianteeman** [comment]
I did a PR that would have fixed this. The biggest issue is that the links are called here and here. I was told to close it and @zero-24 recreated it for J3 and now it is waiting to be merged upstream.

**Comment by brianteeman** [comment]
see #34596 and #34595

**Comment by chmst** [comment]
waiting and closing this for now
